bitnami/aspnet-core/templates/deployment.yaml

```diff
@@ -29,6 +29,7 @@ spec:
     spec:
       {{- include "aspnet-core.imagePullSecrets" . | nindent 6 }}
       automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
```

Comment:
I guess you should remove this line now (and add it to the sa if it's not there already), isn't it?

Reply:
Done